Tanasee/Tanasee1.xml

```diff
@@ -67,15 +740,300 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <langUsage>
                 <language ident="en">English</language>
+                <language ident="am">Amharic</language>
+                <language ident="gez">Gəʿəz</language>
             </langUsage>
         </profileDesc>
         <revisionDesc>
             <change who="PL" when="2021-02-09">Created XML record from table </change>
+            <change when="2023-07-20" who="CH">Added contents and physical description.</change>
         </revisionDesc>
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
         <body>
-            <ab/>
+            <div type="edition" xml:lang="gez" corresp="#a1">
```

Comment:
Probably this encoding is correct, but I believe it would also be correct (and easier for the readers) to include the text of all additions directly in each addition element in `<q>`

Reply:
OK. I will remove all div to the addition elements.